BenchmarkPhasing/BenchmarkPhasing.wdl

```diff
@@ -0,0 +1,162 @@
+            call_vcf=call_vcf,
+            call_vcf_index=call_vcf_index,
+            call_sample_name=call_sample_name,
+            experiment=experiment
+    }
+
+    if (defined(interval_beds)) {
+        call EvaluateRegions {
+            input:
+                switch_errors_bed=WhatsHapCompare.switch_errors_bed,
+                call_sample_name=call_sample_name,
+                experiment=experiment,
+                interval_beds=select_first([interval_beds]),
+                interval_bed_labels=select_first([interval_bed_labels])
+        }
+    }
+
+    output {
+        File whatshap_eval = WhatsHapCompare.eval
+        File switch_errors_bed = WhatsHapCompare.switch_errors_bed
+        File longest_blocks = WhatsHapCompare.longest_blocks
+        File jaccard_table = select_first([EvaluateRegions.jaccard_table, ""])
+    }
+}
+
+task WhatsHapCompare {
+    input {
+        File base_vcf
+        File base_vcf_index
+        String? base_sample_name
+
+        File call_vcf
+        File call_vcf_index
+        String? call_sample_name
+
+        String? experiment
+
+        Boolean only_snvs = false    # Toggle to only consider phasing for SNVs
+
+        # Runtime arguments
+        Int disk_size = 2 * ceil(size(base_vcf, "GB") + size(call_vcf, "GB")) + 50
+        Int cpu = 4
+        Int memory_ram = 16
+    }
+
+    command <<<
+        set -xue
+
+        whatshap compare \
+        --names ~{default="truth" base_sample_name},~{default="callset" call_sample_name} \
+        --tsv-pairwise eval.tsv \
+        --ignore-sample-name \
+        ~{true="--only-snvs" false="" only_snvs} \
+        --switch-error-bed switch_errors.bed \
+        --longest-block-tsv longest_blocks.tsv \
+        ~{base_vcf} ~{call_vcf}
+
+        echo "Experiment" > exp_column.txt
+        yes ~{experiment} | head -n $(cat eval.tsv | wc -l) | sed -e '1d' >> exp_column.txt
+        paste exp_column.txt eval.tsv > final-eval.tsv
+
+        echo "Experiment" > exp_column.txt
+        yes ~{experiment} | head -n $(cat longest_blocks.tsv | wc -l) | sed -e '1d' >> exp_column.txt
+        paste exp_column.txt longest_blocks.tsv > final-longest_blocks.tsv
+    >>>
+
+    runtime {
+        docker: "us.gcr.io/broad-dsde-methods/whatshap:v1"
+        disks: "local-disk " + disk_size + " HDD"
+        cpu: cpu
+        memory: memory_ram + " GB"
+    }
+
+    output {
+        File eval = "final-eval.tsv"
+        File switch_errors_bed = "switch_errors.bed"
+        File longest_blocks = "final-longest_blocks.tsv"
+    }
+}
+
+task EvaluateRegions {
+    input {
+        File switch_errors_bed
+
+        String? call_sample_name
+        String? experiment
+
+        Array[File] interval_beds
+        Array[String] interval_bed_labels
+
+        # Runtime arguments
+        Int disk_size = 2 * ceil(size(switch_errors_bed, "GB") + size(interval_beds, "GB")) + 50
+        Int cpu = 4
+        Int memory_ram = 16
+    }
+
+    command <<<
```

Comment:
This script is complicated enough that I feel it deserves a few line blurb about what's going on, and possibly a little verbal unpacking of the various pasting and sed'n.  Props on using `yes(1)` with a command line argument, I think that's the first time I've seen that one lol.